README.md

```diff
@@ -169,8 +169,48 @@ bazel build --enable_runfiles build_pip_pkg
 TF_NEED_CUDA=1 bazel-bin/build_pip_pkg artifacts
 pip install artifacts/tensorflow_recommenders_addons_gpu-*.whl
 ```
+##### Apple Silicon Support
+**TFRA** Apple Silicon (M1) version requires:
+
+- macOS 12.0.0+
+- Python 3.9
```

Comment:
We should provide release for py3.8, for tensorflow-macos has py38 and py39 release: https://pypi.org/project/tensorflow-macos/2.5.0/#files

Reply:
Added Python 3.8, but I need to test the TFRA build with Python 3.8 first. The testing may take 1-2 days.